webview/src/setup/components/CliIncompatible.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { MIN_CLI_VERSION } from 'dvc/src/cli/dvc/contract'
+import { EmptyState } from '../../shared/components/emptyState/EmptyState'
+
+export const CliIncompatible: React.FC = () => (
+  <EmptyState>
+    <div>
+      <h1>DVC is incompatible</h1>
+      <p>The located CLI is incompatible with the extension</p>
+      <p>The minimum version is {MIN_CLI_VERSION}</p>
+      <p>Please update your install and try again</p>
```

Comment:
Would it be a good idea to add a button here that installs the latest version of DVC? Like we have in `CliUnavailable`?

Reply:
The idea here is that this person should not be going through onboarding if they already have the CLI installed and the version is below the minimum required version. There are too many combinations that we would need to cater for (e.g what if CLI is installed globally). This patch at least avoids us showing something confusing.